src/main/java/tr/com/orioninc/laborant/service/AdminService.java

```diff
@@ -40,24 +46,26 @@ public  String addNewLab(String labName, String userName, String password, Strin
 
     public List<Lab> getAllLabs(){
         List<Lab> allLabs = labRepo.findAll();
+        log.info("[getAllLabs] - Getting all labs from the database... \n {}", allLabs);
```

Comment:
Printing all labs is redundant. We already return the output.

Reply:
done